gadjid/src/graph_loading/edgelist.rs

```diff
@@ -76,15 +78,12 @@ where
 /// associated with this trait, it is purely for strong type checking.
 pub trait IterationLayoutTag: Sized {}
 
-/// Implementation of [`IterationLayoutTag`] for row-major order. Fed as type parameter when
-/// constructing [`Edgelist`]. This indicates that the edgelist returns edges in a row-major order.
 /// This indicates that the outermost index is the row, and the innermost index is the column. The
-/// column index will vary the fastest. The iterator will yield triples of `(row, column, value)`.
+/// column index will vary the fastest. The iterator will yield triples of `(row, column, edge_type)`.
```

Comment:
For clarity, maybe change
outermost → outer
innermost → inner
as there is only two indices, the outer/first and the inner/second.